### ISSUE TITLE
Blender 4.3.0 has strange normal issues with the Cycles renderer

### ISSUE THREAD

**Issue by ailenneko** [QuestionableM/SM-Converter#7]
![5e1387929dc0e1e96315b1cec125b423](https://github.com/user-attachments/assets/2a7f39d6-c1c6-4528-959a-e4e665cee666)
The above image is the result under the EEVEE (Next) renderer.

![db7e71fd140c987dcf6a2cf629d402aa](https://github.com/user-attachments/assets/18993664-f937-45e5-a41c-15f5e4712b8a)
The above image is the result under the Cycles renderer.

I'm using Blender version 4.3.0 and I can confirm that version 4.2.3 is fine. I suspect there are some minor issues with the OBJ format, as the same problem happened in a previous version of Blender. I'm not sure if this is a problem with Blender or an issue with the plugin generating the OBJ format.

**Comment by QuestionableM** [comment]
Looks like the latest version of Blender might've changed the node names again. I'll take a look at it when i have some free time.

Thanks for your report though.

**Comment by QuestionableM** [comment]
It looks like it's not the Plugins fault that the rendering is messed up now. It looks like Blender does some additional post-processing on the .obj which breaks the normals.

**Comment by QuestionableM** [comment]
Yeah ok. All Blender versions starting from 4.3.0 automatically merge the vertices with the same coordinates, which breaks everything. And there seems to be no way of disabling that.

**Comment by QuestionableM** [comment]
@ailenneko I've pushed a a [new prerelease](https://github.com/QuestionableM/SM-Converter/releases/tag/v1.2.2-pre) which added an aforementioned option. Will close the issue once i'll be assured that the issue is solved.
